src/Marten.Testing/Schema/DuplicatedFieldTests.cs

```diff
@@ -135,6 +142,32 @@ public void pg_type_is_used_for_string_list(string pgType, string expectedUpdate
             field.DbType.ShouldBe(NpgsqlDbType.Array | NpgsqlDbType.Text);
         }
 
+        [Theory]
+        [InlineData(null, "date = public.mt_immutable_timestamp(data ->> 'Date')")]
+        [InlineData("myergen", "date = myergen.mt_immutable_timestamp(data ->> 'Date')")]
```

Comment:
`myergen` looks like a very cryptic name for schema. Suggest to change it.

Reply:
Haha, it means "meh", and it's just something I always use without thinking - I just checked and it seems I've used it a [couple](https://github.com/JasperFx/marten/blob/0ccc54748849236c49ccb38fa3f1984b4a2f6efe/src/Marten.Testing/Bugs/Bug_980_bulk_insert_with_subclass.cs#L24) of [times ](https://github.com/JasperFx/marten/blob/0ccc54748849236c49ccb38fa3f1984b4a2f6efe/src/Marten.Testing/Bugs/Bug_899_operations_out_of_order_when_types_use_inheritance.cs#L24) before in previous PRs!

I don't mind changing it if you want though?

Reply:
Alright, we can leave it "as is".